dataactbroker/scripts/initialize.py

```diff
@@ -53,17 +53,13 @@ def createAdmin():
     """Create initial admin user."""
     adminEmail = CONFIG_BROKER['admin_email']
     adminPass = CONFIG_BROKER['admin_password']
-    userDb = UserHandler()
-    try:
-        user = userDb.getUserByEmail(adminEmail)
-    except ResponseException as e:
-
-        if type(e.wrappedException) is NoResultFound:
-            userDb.createUserWithPassword(
-                adminEmail, adminPass, Bcrypt(), permission=2)
-            user = userDb.getUserByEmail(adminEmail)
-            userDb.addUserInfo(user, "Admin", "SYS", "System Admin")
-    userDb.close()
+    with databaseSession() as sess:
+        user = sess.query(User).filter(User.email==adminEmail).one_or_none()
+        sess.close()
+    if not user:
+        user = createUserWithPassword(
+            adminEmail, adminPass, Bcrypt(), permission=2)
```

Comment:
Good point. Since the `createUserWithPassword` is already set up to hard-code the agency code as `CGAC`, I'll update that function to set the name and title to something appropriately admin-y sounding.